src/Teacher.js

```diff
@@ -14,7 +14,7 @@ function AssessmentToHTMLStrTeacherGrade()
     <div class="mainBox">
         <div class="text02Box"> 
             <form action="#">
-                <label for="lang">Question type:</label>
+                <label for="lang">Assessment:</label>
```

Comment:
This should not be a label but a text field to allow the teacher to choose the name for the assessment they wish and not force them into picking a name between the few allowed options.

Also, can you please change "Assessment" to "Assessment Name" please?